### PR TITLE
Fixing JS search 

### PR DESCRIPTION
**Pull request by mmatera**
This PR fixes the issue with the search bar.

### CODE REVIEW

File: mathics_django/web/media/js/doc.js [hunk 1]
```diff
@@ -61,7 +60,6 @@ function loadDoc(page) {
 
 function showDoc() {
         const docLink = document.getElementById('doclink');
-        loadDoc('');
```
Comment (mmatera) [inline mathics_django/web/media/js/doc.js:64]: The problem with reloading the documentation page here is that it breaks the search bar.

File: mathics_django/web/media/js/doc.js [hunk 2]
```diff
@@ -98,6 +96,7 @@ function toggleDoc() {
 	if (document.getElementById('doc').style.display !== 'none') {
 		hideDoc();
 	} else {
+	        loadDoc("/");
```
Comment (mmatera) [inline mathics_django/web/media/js/doc.js:99]: here do not seem to interfere

**Comment by rocky** [comment]
For me this has no effect - broken either way.

**Comment by rocky** [comment]
LGTM - With recent merge into this branch things work for me.

**Pull request merged**